### ISSUE TITLE
freerdp copy&paste doesn't appear to work between two linux machines

### ISSUE THREAD

**Issue by catalyst1987** [FreeRDP/Remmina#1217]
remmina - Version 1.2.0-rcgit-18 (git rcgit-18)
This is FreeRDP version 2.0.0~dev (git n/a)

Host client is Ubuntu 16.04 Unity 7
Guest is Ubuntu Mate 16.04

When I try to copy and paste from host to guest it does not work.  Not sure what I am doing wrong or if it is possible for copy&paste using remmina and freerdp between two linux hosts.

**Comment by ks7000** [comment]
Great software, Remmina is very good!

I am using Ubuntu 16.04 LTS 64 bits with Remmina 1.2.0-rcgit-21 (git rcgit-21), always with RDP at Windows 7 machines and copy/cut/paste neither works between machines.

**Comment by antenore** [comment]
Can you test with xfreerdp?

**Comment by ks7000** [comment]
Terrific! Thanks! I am reading
https://github.com/FreeRDP/FreeRDP/wiki/CommandLineInterface
for "New style options", at glance «+clipboard : redirect clipboard»
I am testing it!

**Comment by antenore** [comment]
Yes but please don't fall in love with it and come back to Remmina, Remmina is forever. ;-)

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @antenore @catalyst1987 @ks7000
